tests/integration/defs/disaggregated/test_disaggregated.py

```
@@ -2602,6 +2638,375 @@ async def run_bursts():
+                        pass
+
+            if not should_cancel:
+                if profile.structured_output_schema is not None:
+                    try:
+                        json.loads("".join(assembled))
+                        result["json_valid"] = True
+                    except json.JSONDecodeError:
+                        result["json_valid"] = False
+                result["success"] = True
+    except Exception:
+        pass  # connection abort on cancel is expected
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)